lib/listener-tracker.js

```diff
@@ -175,9 +206,14 @@ SubscriptionTracker.prototype = {
         }
 
         if (!wrapper) {
-            wrapper = new EventEmitterWrapper(target);
+            if (isNonEventEmitter(target)) {
+              nonEE = new EventEmitterAdapter(target);
+              destroyEvent = DOM_NODE_REMVOED_FROM_DOCUMENT;
```

Comment:
I'm debating if we really want to equate the `DOMNodeRemovedFromDocument` event with the `destroy` event. Maybe this isn't necessary? In most cases this will be used with `subscribeTo(window)` and `window` will never be removed from the DOM. I'm leaning towards not adding destroy event listener for DOM nodes. Thoughts?